vuepress/docs/next/docs/curate/bundle-versions-faq.md

```diff
@@ -1,77 +1,57 @@
-# Bundle versions and updates - FAQ
+---
+sidebarDepth: 1
+---
 
-## Does an Entando Bundle support versioning?
-A bundle, as discussed in the [overview](../compose/local-hub-overview.md), is a package containing one or more components.
-As in many other packaging systems, an Entando Bundle supports versioning, allowing developers to create and release improvements of their package over time.
+# Bundle Versions and Updates - FAQ
 
-## How is a bundle version defined?
+## Support
 
-In the repository, the bundle custom resource comes with a set of `tags`, each one representing a version of the bundle.
-Here an example for a CMS quickstart bundle
-```
-apiVersion: entando.org/v1
-kind: EntandoDeBundle
-metadata:
-    name: cms-quickstart-bundle
-spec:
-  details:
-    name: Entando Example CMS Bundle
-    description: Example Bundle containing CMS components for the Quickstart experience
-  tags:
-    - integrity: ''
-      shasum: ''
-      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
-      version: v0.0.1
-    - integrity: ''
-      shasum: ''
-      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
-      version: v0.0.2
-```
+### 1. Does the Entando Platform support bundle versioning?
+A bundle is an independent package containing one or more components. 
+As in other packaging systems, the [Entando Component Registry](../compose/local-hub-overview.md)(ECR) supports bundle versioning, allowing developers to create and release updates of their package over time.
 
-As you can see from the code above, the bundle `cms-quickstart-bundle` has 2 versions available.
-Each version in the `tags` object must correspond a tag in the Git repository provided in the `tarball` field.
+## How-Tos
+### 1. How do I create a new version of a bundle?
+To release new versions of your bundle after changes have been made, edit the `entando.json` with the new version number for the bundle, then pack and publish your images to Docker. Docker will provide tags to update the bundle version. Once you deploy and install the bundle, the new version number will appear in the App Builder and automatically update in your apps. 
```

Comment:
Minor, remove "and automatically update in your apps". It's not the version number that automatically updates. And, technically, the bundle update isn't automatic either since someone/something has to perform the install in the UI or via CLI.